_studio/mfx_lib/encode_hw/h264/src/mfx_h264_encode_hw_utils_new.cpp

```diff
@@ -2473,10 +2473,29 @@ mfxStatus MfxHwH264Encode::CopyRawSurfaceToVideoMemory(
 
     return MFX_ERR_NONE;
 }
+bool MfxHwH264Encode::IsFrameToSkip(DdiTask&  task, MfxFrameAllocResponse & poolRec, std::vector<mfxU32> fo, bool bSWBRC)
+{
+    if (task.m_isSkipped)
+        return true;
+
+    mfxI32 fid = task.m_fid[0];
+    if (task.m_list1[fid].Size() && bSWBRC && (task.GetFrameType() & MFX_FRAMETYPE_B))
+    {
+        mfxU8 ind = task.m_list1[fid][0] & 127;
+        if (ind < 15 )
```

Comment:
why 15? Can this be used as some macro or const int with the reasonable name instead of magic number?